src/encoded/static/components/genome_browser.js

```diff
@@ -597,6 +599,8 @@ const TrackLabel = ({ file, label, supplementalShortLabel, long }) => {
         ]).join(', ');
     }
 
+    const originBatchesList = file.origin_batches ? `batch${file.origin_batches.length > 1 ? 'es' : ''}: ${file.origin_batches.map((batch) => batch.split('/')[2]).join(', ')}` : '';
```

Comment:
This code is fine. Some feel though it makes code unnecessarily to have nested ternary operators.

Reply:
thanks.
I am not an expert in JS, but a one-liner looked better than 3 lines.

Reply:
For future considerations I think 3 lines is more comprehensible unless you break it up with <returns>